notebooks/Roberts_Mock_data.ipynb

```diff
@@ -0,0 +1,1113 @@
+        "    \"Kansas\", \"Kentucky\", \"Louisiana\", \"Massachusetts\", \"Maryland\", \"Maine\",  \n",
+        "    \"Minnesota\", \"Missouri\", \"Mississippi\", \"Montana\", \"North Carolina\", \n",
+        "    \"North Dakota\", \"Nebraska\", \"New Hampshire\", \"New Jersey\", \"New Mexico\", \n",
+        "    \"Nevada\", \"New York\", \"Ohio\", \"Oklahoma\", \"Oregon\", \"Pennsylvania\", \n",
+        "    \"Puerto Rico\", \"Rhode Island\", \"South Carolina\", \"South Dakota\", \n",
+        "    \"Tennessee\", \"Texas\", \"Utah\", \"Virginia\", \"Virgin Islands\", \"Vermont\", \n",
+        "    \"Washington\", \"Wisconsin\", \"West Virginia\", \"Wyoming\", \"Michigan\",\n",
+        ")\n",
+        "\n",
+        "convictions = (\n",
+        "    \"Misdemeanors\", \"Felony\", \"Wobbler\",\n",
+        ")\n",
+        "\n",
+        "skills = (\n",
+        "    \"HTML\", \"CSS\", \"JavaScript\", \"Python\", \"Ruby\",\n",
+        "    \"C++\", \"Axios\", \"React\", \"Node.JS\", \"C#\", \"Django\",\n",
+        "    \"MongoDB\", \"SQL\", \"PostgreSQL\", \"NoSQL\", \"PHP\",\n",
+        "    \"Angular\", \"Microsoft Azure\", \"IOS\", \"Android\"\n",
+        ")\n",
+        "\n",
+        "time_zones = (\n",
+        "    \"Pacific\", \"Mountain\", \"Central\", \"Eastern\"\n",
+        "\n",
+        ")\n",
+        "\n",
+        "section_of_day = (\n",
+        "    \"Morning: 6AM - 12PM\", \"Afternoon: 12PM - 6PM\", \"Evening: 6PM - 12AM\"\n",
+        ")\n",
+        "\n",
+        "purpose = (\n",
+        "    \"Technical\", \"Career Preparation\"\n",
+        ")\n",
+        "\n",
+        "\n",
+        "\"\"\"\n",
+        "Time_of_day is split up into 6 hour increments. \n",
+        "Morning is 6AM - 12PM\n",
+        "Afternoon is 12PM - 6PM\n",
+        "Evening is 6PM - 12AM\n",
+        "\"\"\"\n",
+        "\n",
+        "time_of_day = (\n",
+        "    \"Morning\", \"Afternoon\", \"Evening\"\n",
+        ")\n",
+        "\n",
+        "goals = (\n",
+        "    \"I want to get a job in tech\", \n",
+        "    \"I want to hone my skills\", \n",
+        "    \"I want to better prepare for the interview process\", \n",
+        "    \"I want to network\"\n",
+        ")"
+      ],
+      "metadata": {
+        "id": "1WmRh0a3Zc6O"
+      },
+      "execution_count": 210,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Now that we have some source data out of the way, we can get to the fun part, creating the Mentor and Mentee classes that we will use to populate our DataFrame"
+      ],
+      "metadata": {
+        "id": "We530RIhtbou"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 221,
+      "metadata": {
+        "id": "_UFRJfn6UY8f"
+      },
+      "outputs": [],
+      "source": [
+        "import random as r\n",
+        "import pandas as pd\n",
+        "\n",
+        "\n",
+        "class Mentee:\n",
```

Comment:
Its clearer when looking at the notebook but I like that you group like attributes with like attributes. It makes it easier to read.